src/index.css

```diff
@@ -0,0 +1,46 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+
+* {
```

Comment:
use the equalizer as a css reset instead of creating something new. you can find it in the tech guides under the frontend and layout folders.

http://cssreset.com/what-is-a-css-reset/